src/containers/DayEntries.js

```diff
@@ -0,0 +1,56 @@
+import {fetchEntries} from '../actions/entriesActions'
+import Entry from '../presentational/entry/Entry';
+import {connect} from 'react-redux';
+import React, {Component} from 'react'
+import Util from '../Util';
+
+class DayEntries extends Component {
+    componentDidMount() {
+        this.props.fetchEntries(Util.dateToDatestamp(this.props.from), Util.dateToDatestamp(this.props.to));
+    }
+
+    // https://github.com/reduxjs/redux/issues/448
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.from !== this.props.from && nextProps.to !== this.props.to) {
+            this.props.fetchEntries(Util.dateToDatestamp(nextProps.from), 
```

Comment:
this is to responds to changes in the date period (triggered by Date Navigator component), we reload the entries if the date period changed.